wordpress-mobile/WordPress-iOS
# Reload all data instead of section only.

On pages the featured image while has the same tag is
not on the same position so reloading the section make
the code to crash.

Fixes #14098 

To test:
  - Start the app
  - Create a **page** 
  - Open page settings and add a featured image to it.
  - Tap on the featured image
 - Tap on remove
 - Check that it doesn't crash.

 - Repeat the steps above but now using a **post** and see if all is ok there too.

PR submission checklist:
